### ISSUE TITLE
create_device fails in multiple ways

### ISSUE THREAD

**Issue by kke** [packethost/packet-rb#6]
I'm having problems trying to create a device through the API using this gem.

```
/usr/local/opt/rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/packethost-0.0.5/lib/packet/client.rb:59:in `fail_on_error': {"errors"=>["#<Packet::Plan:0x007fb5e3946df8> is not a valid plan"]} (Packet::Error)
        from /usr/local/opt/rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/packethost-0.0.5/lib/packet/client.rb:24:in `block (2 levels) in <class:Client>'
```

Example code:

```ruby
require 'packet'

API_KEY=''
PROJECT_ID=''
FACILITY_CODE=''
OS_SLUG=''
PLAN_SLUG=''

# Monkeypatch create_device to accept project_id (since the method in the gem doesn't seem to work?)
module Packet
  class Client
    module Devices
      def create_device(project_id, device)
        post("projects/#{project_id}/devices", device.to_hash).tap do |response|
          device.update_attributes(response.body)
        end
      end
    end
  end
end

def abort(msg)
  puts msg
  exit 1
end

def client
  @client ||= ::Packet::Client.new(API_KEY)
end

def find_project(project_id)
  client.list_projects.find{|project| project.id == project_id}
end

def find_facility(facility_code)
  client.list_facilities.find{|f| f.code == facility_code}
end

def find_os(os_code)
  client.list_operating_systems.find{|os| os.slug == os_code}
end

def find_plan(plan_code)
  client.list_plans.find{|plan| plan.slug == plan_code}
end

abort('Project does not exist in Packet') unless project = find_project(PROJECT_ID)
abort('Facility does not exist in Packet') unless facility = find_facility(FACILITY_CODE)
abort('Operating system does not exist in Packet') unless os = find_os(OS_SLUG)
abort('Device type does not exist in Packet') unless plan = find_plan(PLAN_SLUG)

device = ::Packet::Device.new(
    hostname: 'testing 123',
    facility: facility.to_hash, # by passing an id or a Packet::Facility will give unknown method each_pair
    operating_system: os.to_hash, # same with this one
    plan: plan.to_hash,
    billing_cycle: 'hourly',
    locked: true
)

response = client.create_device(project.id, device)

```

**Comment by kke** [comment]
I created a PR to fix this 
#7

**Comment by kke** [comment]
Fixed in 0.0.6
